v5/core/utils.go

```diff
@@ -182,6 +188,20 @@ func GetCurrentTime() int64 {
 	return time.Now().Unix()
 }
 
+// ParseDate parses and returns a strfmt Date pointer
+func ParseDate(dateString string) *strfmt.Date {
+	formattedTime, _ := time.Parse("2006-01-02", dateString)
+	formattedDate := strfmt.Date(formattedTime)
+	return &formattedDate
+}
+
+// ParseDateTime parses and returns a strfmt DateTime pointer
+func ParseDateTime(dateString string) *strfmt.DateTime {
+	formattedTime, _ := time.Parse(time.RFC3339, dateString)
```

Comment:
You should use `strfmt.ParseDateTime` here.  It has already been set up to handle all the date-time formats we need.

Reply:
Updated to use `strfmt.ParseDateTime` with error handling!